src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,13 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int INITIAL_CAPACITY = 10;
+    private static final int KEY_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private int incrementedIndex = 0;
+    private Object[][] storage;
+
+    public StorageImpl() {
+        storage = new Object[INITIAL_CAPACITY][INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
+        Object[] result = isContains(key);
+        if ((boolean) result[0] == true) {
+            storage[(int) result[1]][VALUE_INDEX] = value;
+        } else {
+            storage[incrementedIndex][KEY_INDEX] = key;
+            storage[incrementedIndex][VALUE_INDEX] = value;
+            incrementedIndex++;
+        }
     }
 
     @Override
     public V get(K key) {
-        return null;
+        Object[] result = isContains(key);
+        return (V) result[2];
+    }
+
+    private Object[] isContains(K key) {
```

Comment:
```suggestion
    private int indexOf(K key) {
```
I would suggest you to refactor this method

let's iterate over the `keys` of your storage (you can use `incrementedIndex` for the actual length by the way)
if we find the matching key - we return the index
if not - return `-1`

Reply:
Greate idea, I understud you - But I will to beg you accept my code, cosa Im in ArraList implementation now